simplivity/resources/datastores.py

```diff
@@ -176,3 +176,23 @@ def resize(self, size, timeout=-1):
         self.data = datastore_obj.data
 
         return self
+
+    def set_policy(self, policy_id, timeout=-1):
```

Comment:
this should accept the policy object or the policy name, and to the correct conversion within the method.

please see set_policy within virtual_machines for an example

Reply:
Updated the code as suggested